tests/conftest.py

```diff
@@ -152,7 +152,7 @@ def rnd_str(n):
     return ''.join(np.random.choice(list('abcdefghijklmnopqrstuvwxyz'), n))
 
 
-@pytest.fixture(autouse=True)
+@pytest.fixture(scope='session', autouse=True)
```

Comment:
I thought scope='session' was the default...

Reply:
I think the default is scoped to a function. Session scope is the "biggest" one.